assets/css/custom-props/_elixir.css

```diff
@@ -8,11 +8,11 @@
   --searchBarFocusColor:  #8E7CE6;
   --searchBarBorderColor: rgba(142, 124, 230, 0.25);
 
-  --linksNoUnderline:            var(--mainDark);
-  --linksNoUnderlineVisited:     var(--mainDarkest);
+  --link-color:            var(--mainDark);
+  --link-visited-color:     var(--mainDarkest);
 }
 
 body.dark {
-  --linksNoUnderline:            var(--mainLightest);
-  --linksNoUnderlineVisited:     var(--mainLight);
+  --link-color:            var(--mainLightest);
+  --link-visited-color:     var(--mainLight);
```

Comment:
Sorry, to be clear, I was thinking we should move these out of the language custom props and use the same for both languages (so they would go to the general prop definitions). I think having both languages use the same colors for links would be a huge plus for overall ecosystem consistency. Thoughts?